rl_algorithms/common/networks/base_network.py

```diff
@@ -46,11 +47,12 @@ def forward_(self, x: torch.Tensor, n_tau_samples: int = None):
         return x
 
 
-def calculate_fc_input_size(state_dim: tuple, cnn: ConfigDict):
+def calculate_fc_input_size(state_dim: tuple, backbone_cfg: ConfigDict):
     """calculate fc input size according to the shape of cnn"""
```

Comment:
- Capitalize the first letter
- Put a period

Reply:
resolved in commit 2cad1a7!!